### PR TITLE
Change BrewRenderer background in Heroku test deployments

### PR DESCRIPTION
**Pull request by G-Ambatte**
This PR modifies the BrewRenderer to change the default background when the `HEROKU_APP_NAME` variable exists. This variable only exists when created during the PR deployment process in Heroku apps, or when specifically set on a Development machine.

The main purpose of this PR is to create a visual indication that someone is working on a deployment, NOT the main website. This is primarily intended to be a Dev Experience QoL improvement, which is why I've marked it as a low priority change.

### CODE REVIEW

**Comment by calculuschild** [comment]
What issue is this trying to solve?

**Comment by 5e-Cleric** [comment]
> What issue is this trying to solve?

This is trying to help us in the development, so we don't confuse local vs deployment vs live. Minor inconveniences.

**Comment by dbolack-ab** [comment]
Can we get a deployment on this one to validate?

**Review by dbolack-ab — APPROVED** [review]
Looks good. Need to see the deployment.

File: client/homebrew/brewRenderer/brewRenderer.jsx [hunk 1]
```diff
@@ -188,6 +188,12 @@ const BrewRenderer = (props)=>{
 		}));
 	};
 
+	const styleObject = { height: state.height };
```
Comment (5e-Cleric) [inline client/homebrew/brewRenderer/brewRenderer.jsx:191]: height:100vh works fine here, i tested in another PR. This works too, but is less readable.

**Review by dbolack-ab — APPROVED** [review]
Looks good, deployment works great for the intended use.

**Comment by calculuschild** [comment]
@G-Ambatte Can you address 5e-Cleric's open comment and then I can merge this.

**Comment by G-Ambatte** [comment]
> @G-Ambatte Can you address 5e-Cleric's open comment and resolve conflicts, and then I can merge this.

I've implemented the suggested change... which led me to investigate if `state.height` was actually being used anywhere after the change. As best I can determine it was not, so I have also removed the now obsolete state and event listeners from BrewRenderer.

**Review by calculuschild — APPROVED** [review]

**Comment by calculuschild** [comment]
Nice added cleanup. Merging!

**Pull request merged**